src/map/map.cpp

```diff
@@ -188,6 +188,12 @@ int console = 0;
 int enable_spy = 0; //To enable/disable @spy commands, which consume too much cpu time when sending packets. [Skotlex]
 int enable_grf = 0;	//To enable/disable reading maps from GRF files, bypassing mapcache [blackhole89]
 
+#ifdef MAP_GENERATOR
+struct generator_options {
+	bool gen_navi;
```

Comment:
```suggestion
	bool navi;
```
Just a thought, really. Since the struct is prefixed with "gen_" do we also want to prefix each of the options with the same thing?

Reply:
Makes sense